fetch_data.py

```diff
@@ -200,17 +227,21 @@ def deserialize_order_items(region, redo_urls, func):
     return deserialized_results, redo_urls
 
 
+def parse_history_results(results, histories):
+    for result in results:
+        result_item_id = int(result.url.split("=")[-1])
+        item_history = json.loads(result.text)
+        histories[result_item_id] = item_history
+
+
 def deserialize_history_chunk(history_urls, histories):
     for history_chunk in history_urls:
         results, redo_urls, error_timer = futures_results(create_history_futures(history_chunk))
```

Comment:
Does the results of redo_URLs get added to the history?

Seems like it, but would like to confirm.

Reply:
And if so, think about how to not add the bad re-do data in here.

Reply:
Resolving this as #32 addresses issue.